build/pluginctl/main.go

```diff
@@ -85,6 +99,23 @@ func getClient() (*model.Client4, error) {
 	return nil, errors.New("one of MM_ADMIN_TOKEN or MM_ADMIN_USERNAME/MM_ADMIN_PASSWORD must be defined")
 }
 
+func getUnixClient(socketPath string) (*model.Client4, error) {
```

Comment:
We're using `error` as a proxy for "found socket" here, but we never actually /check/ the value of the returned error. Should we test the error values to distinguish "not running" vs. "tried but failed for other reasons"? Or should we just return `*model.Client4, bool` to make this more explicit?